### ISSUE TITLE
[Feature request] have columns inside a list field in a blueprint

### ISSUE THREAD

**Issue by dimitrilongo** [getgrav/grav-plugin-admin#1006]
I think it will be useful to have colums working in list field, make sense with many fields inside list items (visual)

See these threads :
https://github.com/getgrav/grav-plugin-admin/issues/867

https://getgrav.org/forum#!/general:can-you-have-columns-inside

**Comment by OleVik** [comment]
I think this makes sense for declaring cleaner interfaces. That is, separating "shorter" fields like buttons for binary settings or selects into columns is appropriate when they are taxonomically related.

Consider a plugin with three distinct functionalities: Structuring content (HTML), including styles (varied CSS), including scripts (varied JS). They could just as easily be vertically separated using a [Spacer](https://learn.getgrav.org/forms/forms/fields-available#the-spacer-field) or [Section](https://learn.getgrav.org/forms/blueprints/fields-available#the-section-field), but this inevitably leads to a tall interface which requires a fair amount of scrolling for a plugin with a high degree of specificity. Separating the settings into ordered categories of fields can make for a shorter interface.

It is, of course, not appropriate to blithely declare everything in columns as this will break usability and accessibility if fields are comparatively long (horizontally). On smaller devices, however, the columns could easily be styled to break into the equivalent of separating with Spacers so UX is maintained even on mobile.

**Comment by rhukster** [comment]
hey @mahagr did you resolve something similar to this with G5 tabs?

**Comment by mahagr** [comment]
Yes, I solved this for Gantry, though it had (and still has) some teething issues.

**Comment by hexplor** [comment]
This would be super duper useful :) :) :)

**Comment by Gsoaresmota** [comment]
Do you know more information about this theme?

Thank you.
